README.md

```diff
@@ -86,6 +86,7 @@ Emoji | Represents | Links to | Comment
 📦 | Packaging/porting to support a new platform
 🔌 | Plugin/utility libraries | the repo home
 👀 | Reviewed Pull Requests
+🛡️ | Security
```

Comment:
What type of contribution is security exactly? I mean, what would someone do to make a security contribution?

Reply:
Great question! I didn't want to put a description to narrow the field (altho maybe we want that?)

Along the lines of, conducts security reviews, helped identify and reduce security threats of the project. Could even tie into GDPR and privacy

Reply:
Updated readme